Main.py

```diff
@@ -167,10 +169,12 @@ def main(args, seed=None, baked_server_options: Optional[Dict[str, object]] = No
 
     # remove starting inventory from pool items.
     # Because some worlds don't actually create items during create_items this has to be as late as possible.
-    if any(world.start_inventory_from_pool[player].value for player in world.player_ids):
+    if any(hasattr(world.worlds[player].options, "start_inventory_from_pool") for player in world.player_ids):
```

Comment:
I think this change is not fully correct. the old code skips for `None` as well as `{}` , the new code will run for `{}`. Should probably be `any(getattr(..., ..., None) for ...)`